packages/pyright-internal/src/tests/analyser.test.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * symbolNameUtils.test.ts
```

Comment:
Nit: Filename does not match, and by this repo's convention the spelling would be "analyzer" if anything. I'd prefer if it were named "symbolNameUtils.test.ts", however.

Reply:
Good catch! `symbolNameUtils.test.ts` was what I originally though off. Will change it back